### DISCUSSION TITLE
How to disable auth for a http-in node?

### DISCUSSION THREAD

**Discussion by KonstantinKa** [hassio-addons/addon-node-red#1351] [General] [answered]
How i can disable authorization in use node "http-in"?
![image](https://user-images.githubusercontent.com/10378880/169957954-f4be5173-85fc-47e3-a35a-34378e17ea60.png)

**Comment by frenck** [comment, marked as answer]
You can't.

**Reply by KonstantinKa** [reply]
disabled, thanks)
